SRC/CMakeLists.txt

```diff
@@ -53,7 +53,8 @@ set(SCLAUX
    slasd0.f slasd1.f slasd2.f slasd3.f slasd4.f slasd5.f slasd6.f
    slasd7.f slasd8.f slasda.f slasdq.f slasdt.f
    slaset.f slasq1.f slasq2.f slasq3.f slasq4.f slasq5.f slasq6.f
-   slasr.f  slasrt.f slassq.f slasv2.f spttrf.f sstebz.f sstedc.f
```

Comment:
shouldn't the Makefile also be changed?

Reply:
added in commit 8bf331229f9bc1022886eb57eb35ab4bf06938d0